src/components/common/helpers/Helpers.tsx

```diff
@@ -1173,3 +1174,15 @@ export const getAPIOptionsWithTriggerTimeout = (options?: APIOptions): APIOption
 
     return _options
 }
+
+export const getApprovalModalTypeFromURL = (url: string): APPROVAL_MODAL_TYPE => {
+    if (url.includes(APPROVAL_MODAL_TYPE.DEPLOYMENT.toLocaleLowerCase())) {
+        return APPROVAL_MODAL_TYPE.DEPLOYMENT
+    }
+
+    if (url.includes(APPROVAL_MODAL_TYPE.IMAGE_PROMOTION.toLocaleLowerCase())) {
```

Comment:
shouldn't we do .lowercase on the url instead to have the case insensitive check?